stan/math/prim/fun/one_hot_row_vector.hpp

```diff
@@ -0,0 +1,33 @@
+#ifndef STAN_MATH_PRIM_FUN_ONE_HOT_ROW_VECTOR_HPP
+#define STAN_MATH_PRIM_FUN_ONE_HOT_ROW_VECTOR_HPP
+
+#include <stan/math/prim/err.hpp>
+#include <stan/math/prim/fun/Eigen.hpp>
+
+namespace stan {
+namespace math {
+
+/**
+ * Return a row vector with 1 in the k-th position and zero elsewhere
```

Comment:
[optional]
Minor detail, but these are full imperative sentences that should end with periods.